SPEC.md

```diff
@@ -322,9 +346,15 @@ string characters except unescaped `"` and `\`. This includes literal
 multiple lines without behaving like a Newline for [Node](#node) parsing
 purposes.
```

Comment:
I think this sections needs to be reworded: according to the grammar, the string values can no longer "just" have literal newline characters, it needs to be a valid multi-line string. (In fact, literal newline characters are regulated pretty heavily now.)